sis-local/example/README.md

```diff
@@ -0,0 +1,33 @@
+# Example usage of streamlit_in_snowflake
+
+1. Create a secrets.toml file in this repo, as described in the parent directory README
+2. Create and activate a python 3.8 virtual environment, such as with Conda (see below)
+3. Run the following
+
+```shell
+pip install -r requirements.txt
```

Comment:
Isn't this deprecated after we recommend using Conda? Or there should be a 
```
conda activate my_env
```
before that?

Reply:
I wanted to put the minimal example here and the full conda example below, because someone CAN use this without conda and it may be overwhelming if they aren't familiar with it and just want to build a simple app.

The bullet point just above says "you need to create a virtual environment, we recommend doing it with conda".

Reply:
I could also remove the requirements but still have `pip install <link to repo>` if the concern is about having both a requirements file and an environment file... Maybe that would be better. WDYT @blackary ?